vaadin-testbench-core-junit5/src/main/java/com/vaadin/testbench/browser/BrowserExtension.java

```diff
@@ -125,7 +125,7 @@ protected String getHubURL(Class<?> testClass) {
         if (SauceLabsIntegration.isConfiguredForSauceLabs()) {
             return SauceLabsIntegration.getHubUrl();
         } else {
-            return "http://" + getHubHostname(testClass) + ":4444/wd/hub";
+            return String.format("http://%s:%d/wd/hub", getHubHostname(testClass), Parameters.getHubPort();
```

Comment:
```suggestion
            return String.format("http://%s:%d/wd/hub", getHubHostname(testClass), Parameters.getHubPort());
```
Sorry missing the end )